keel-web/src/main/kotlin/com/netflix/spinnaker/keel/rest/DeliveryConfigController.kt

```diff
@@ -84,44 +77,5 @@ class DeliveryConfigController(
     //  in [ExceptionHandler])
     mapOf("status" to "valid")
 
```

Comment:
These are moved to the `ApplicationController` for ease of use in the UI.

Reply:
That will break the API contract and require changes to gate, no?

Reply:
yep, it will, but that's only used in the slack mj response api, right?

Reply:
Well, it was used up until now as the only way to set constraint state _besides_ manual judgement. But since the whole point of the PR is to unblock the new UI to do this, I think we're good.